tests/pytorch/test_cuda_graphs.py

```diff
@@ -232,12 +233,23 @@ def _test_cuda_graphs(
                 )
                 for _ in range(num_layers)
             ]
-        elif module == "layernorm_mlp":
+        elif module == "layernorm_mlp_nocheckpoint":
             modules = [
                 LayerNormMLP(
                     model_config.hidden_size,
                     model_config.hidden_size,
                     params_dtype=dtype,
+                    checkpoint=False,
+                )
+                for _ in range(num_layers)
+            ]
+        elif module == "layernorm_mlp_checkpoint":
```

Comment:
My suspicion is that this test is failing due to RNGs. The cuda graph test as well as LNMLP checkpoint both modify rng, so we should ensure that the correct state is being restored.